vtex-sites/base.store
# feat: Adds Order Summary Component

## What's the purpose of this pull request?
This PR aims to create the CartSummary component following this [jira task](https://vtex-dev.atlassian.net/browse/FSSS-135).

## How it works? 
The `cart/OrderSummary` component was created under the `cart` folder. 
All Cart-related props are retrieved in their parent `CartSidebar`. The checkout button is also generated at the parent to avoid sending a lot of props to the summer component. The button still lacks some behaviors that will be handled later.

## How to test it?
Go to the [preview page](https://preview-253--base.preview.vtex.app), add some items to the cart, then open it. The summary part should be looking like this:
<img width="729" alt="image" src="https://user-images.githubusercontent.com/8497187/150001586-04c248ff-2677-4af5-903d-5a5bc3bb5dde.png">
